libraries/ms-helper/inc/relay_rx.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+// Module for interfacing to CAN for CAN controlled relays.
+//
+// Requires CAN to be initialized.
+//
+// Allows registering a RelayRxHandler that is wrapped by a CANRxHandlerCb which is properly
+// configured to unpack relay messages and handle faults. This RelayRxHandler is expected to execute
+// the modifications to any GPIO pins or other mechanisms to trigger state changes to relays.
+
+#include <stdint.h>
+
+#include "can_msg_defs.h"
+#include "status.h"
+
+// Wrapped by a CANRxHandlerCb.
+typedef StatusCode (*RelayRxHandler)(SystemCanMessage msg_id, uint8_t state, void *context);
+
+typedef struct RelayRxStorage {
+  RelayRxHandler handler;
+  SystemCanMessage msg_id;  // Unused but helpful in debugging.
+  uint8_t curr_state;
+  uint8_t state_bound;
+  void *context;
+} RelayRxStorage;
+
+// Initialize the relay module to use the supplied |relay_storage| of |size| to hold relay
+// configurations.
+StatusCode relay_rx_init(RelayRxStorage *relay_storage, size_t size);
```

Comment:
This should be moved to the non-static paradigm

Reply:
Done.